handler/src/main/java/io/netty/handler/ssl/OpenSslKeyMaterialManager.java

```diff
@@ -74,7 +74,7 @@ void setKeyMaterial(OpenSslEngine engine) throws SSLException {
         }
     }
 
-    void setKeyMaterial(OpenSslEngine engine, String[] keyTypes, X500Principal[] issuer) throws SSLException {
+    void setKeyMaterial(ReleasableOpenSslEngine engine, String[] keyTypes, X500Principal[] issuer) throws SSLException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)